gordo_components/builder/build_model.py

```diff
@@ -48,10 +49,16 @@ def build_model(
         Mapping of the Dataset to initialize, following the same logic as model_config.
     metadata: dict
         Mapping of arbitrary metadata data.
+    cv_mode: str
```

Comment:
Numpy usually writes the 3 explicit choices, like this: https://github.com/numpy/numpy/blob/d9b1e32cb8ef90d6b4a47853241db2a28146a57d/numpy/core/fromnumeric.py#L509

Reply:
But I am also fine with the `str`, especially if the typechecker `mypy` finds it easier.